javascript/javascript3/week1/GiphyApi.js

```diff
@@ -0,0 +1,42 @@
+
+const search = document.getElementById('searchText')
+const resultNumber = document.getElementById('number')
+const button = document.querySelector('button')
+const div = document.getElementById('center')
+
+search.addEventListener('change',startSearch)
+
+
+   function startSearch(){
+      if (search.value){
+      div.innerHTML = 'Please enter number to start search or just press button!'
+      button.addEventListener('click',searchButton)
```

Comment:
I am not sure if you need every time adding the event listener. you can add it in the line 8.